tests/logic.js

```diff
@@ -171,7 +171,7 @@ define([
                 var form = util.loadXML(MOTHER_REF_XML),
                     manager = form._logicManager;
                 assert.deepEqual(manager.caseReferences(),
-                    {load: {"/data/mug": ["parent/edd"]}});
+                    {load: {"/data/mug": ["#case/parent/edd"]}});
```

Comment:
I think that the result of this will be the case summary shows `#case/parent/edd` and it currently shows `parent/edd` which probably isn't intended. That becomes worse once we send user properties as under a child case it could say `#user/property` which doesn't make sense to display